addons/sys_prc343/radio/__PREP__.sqf

```diff
@@ -19,12 +20,15 @@ PREP_MODULE(radio,getState);
 PREP_MODULE(radio,setState);
 PREP_MODULE(radio,getOnOffState);
 PREP_MODULE(radio,setOnOffState);
-PREP_MODULE(radio,getListInfo);
+//PREP_MODULE(radio,getListInfo);
 PREP_MODULE(radio,handleMultipleTransmissions);
 PREP_MODULE(radio,handleBeginTransmission);
 PREP_MODULE(radio,handleEndTransmission);
 PREP_MODULE(radio,handleSignalData);
 PREP_MODULE(radio,handlePTTDown);
 PREP_MODULE(radio,handlePTTUp);
 PREP_MODULE(radio,isExternalAudio);
-PREP_MODULE(radio,getExternalAudioPosition);
\ No newline at end of file
+PREP_MODULE(radio,getExternalAudioPosition);
+
+PREP_MODULE(radio,closeGUI);
+PREP_MODULE(radio,openGUI);
```

Comment:
There is a blank line, you can see on removed line 30 how GitHub displays no newline at end of file.